src/components/flat-table/flat-table-row/flat-table-row.component.tsx

```diff
@@ -23,6 +24,12 @@ import SubRowProvider, { SubRowContext } from "./__internal__/sub-row-provider";
 import { buildPositionMap } from "../__internal__";
 import FlatTableHeadContext from "../flat-table-head/__internal__/flat-table-head.context";
 
+type DesignTokensType = keyof typeof DesignTokens;
+export type BorderRadiusType = Extract<
```

Comment:
Comment: we seem to be re-using this type from the `Box` component so imo it's worth doing the same here. We should probably review whether we want this type to be publicly exposed or not, if we do need to move it internal having them all imported in a consistent way will limit the likelihood we miss one etc

Reply:
I also think we should limit this a bit more, we don't want to support `borderRadiusCirlce` for example. I'm also not convinced we want to support this on every row, so it might be better moving the prop to `FlatTable` and restricting it to the first and last rows etc

Reply:
@Parsium I still think it's worth restricting this a bit more, if we don't then do it later it's a breaking change etc